tests/Process/Builder/ProcessBuilderTest.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Infection\Tests\Process\Listener;
```

Comment:
this namespace is incorrect and leads to failed build https://travis-ci.org/infection/infection/jobs/274783226#L603

It should be `Infection\Tests\Process\Builder` I guess